packages/ckeditor5-theme-lark/theme/ckeditor5-ui/components/dropdown/menu/dropdownmenu.css

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+.ck.ck-dropdown-menu__menu {
+	/* Enable font size inheritance, which allows fluid UI scaling. */
+	font-size: inherit;
```

Comment:
Why do we need that? The scaling looks fine without it, for instance, if changed `--ck-font-size-base` to `18px`. 
<img width="686" alt="image" src="https://github.com/user-attachments/assets/dbafd7ce-954c-4597-8432-9de9c0d090df">

Reply:
I have no idea. I am fine with removing it.